lib/ae_page_objects/node.rb

```diff
@@ -16,10 +16,18 @@ def current_url_without_params
     end
 
     is_loaded do
-      if locator = loaded_locator
-        node.first(*eval_locator(locator)) != nil
-      else
+      locator = eval_locator(loaded_locator)
+      if locator.empty?
```

Comment:
I don't understand this change.  We count it as loaded if there is no locator?

Reply:
Yes, that is maintaining existing behavior, I've just swapped the `if/else` so that rubocop doesn't complain about assigning a variable within an `if` statement.